docs/api-reference/arcgis/arcgis-deck-layer.md

```diff
@@ -0,0 +1,53 @@
+# ArcGISDeckLayer
+
+This class inherits from the ArcGIS [Layer](https://developers.arcgis.com/javascript/latest/api-reference/esri-layers-Layer.html) class and can be added to maps created with the ArcGIS API for JavaScript. Its content is defined by the `deckLayers` property, which is a collection of deck.gl layers.
+
+## Usage
+
+```js
+import {loadArcGISModules} from '@deck.gl/arcgis';
+import {GeoJsonLayer, ArcLayer} from '@deck.gl/layers';
+
+loadArcGISModules([
+  "esri/Map",
+  "esri/views/MapView"
+]).then(([
+  arcGIS,
+  ArcGISMap,
+  MapView
+]) => {
+  const layer = new arcGIS.ArcGISDeckLayer({
+    deckLayers: [
```

Comment:
Is there a reason to call it `deckLayers` instead of just `layers`? Aligning with the `Deck` class' prop names should make it easier to onboard?

Reply:
I had the impression (twice) that `layers` was conflicting with something else. But I wanted to investigate more. I'll have another look.